api/sessiontokenloginprovider.go

```diff
@@ -64,9 +64,8 @@ func (p *sessionTokenLoginProvider) Login(ctx context.Context, caller base.APICa
 	if err == nil {
 		return result, nil
 	}
-
-	if params.ErrCode(err) == params.CodeUnauthorized {
-		// if we fail with an "unauthorized" error, we initiate a
+	if params.ErrCode(err) == params.CodeSessionTokenInvalid {
```

Comment:
```suggestion
	if params.IsCodeSessionTokenInvalid(err) {
```

Can you make a method in `rpc/params/apierror.go` (there are examples of this already). Also I wonder if this should be added to `TranslateWellKnownError` in the same package.

Reply:
Added `params.IsCodeSessionTokenInvalid`.